service-discovery/consul/src/test/java/io/smallrye/stork/servicediscovery/consul/ConsulServiceDiscoveryIT.java

```diff
@@ -64,12 +104,29 @@ void shouldGetServiceFromConsul() throws InterruptedException {
         assertThat(instances.get()).hasSize(1);
         assertThat(instances.get().get(0).getHost()).isEqualTo("example.com");
         assertThat(instances.get().get(0).getPort()).isEqualTo(8406);
+
+        deregisterService(serviceName);
+
+        // When the refresh interval is reached
+        Thread.sleep(65000);
```

Comment:
can you set the expiration in config to 1s or 5s so taht we don't have to wait that long in tests?

Reply:
Done, I had to change the time unit to seconds also

Reply:
I think setting the interval to `5s` should work, without appending `s` before parsing

Reply:
done! can you review Michal?